src/System.Linq.Parallel/src/System/Linq/Parallel/Utils/Sorting.cs

```diff
@@ -65,14 +65,12 @@ internal class SortHelper<TInputOutput, TKey> : SortHelper<TInputOutput>, IDispo
             Contract.Assert(sharedkeys != null);
```

Comment:
No. That was intentional. It's a separate commit. We're only now truly building with Dev14 and it caught this.